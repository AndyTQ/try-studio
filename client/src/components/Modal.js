import React from 'react';
import { withStyles, makeStyles } from '@material-ui/core/styles';
import Button from '@material-ui/core/Button';
import Dialog from '@material-ui/core/Dialog';
import MuiDialogContent from '@material-ui/core/DialogContent';
import MuiDialogActions from '@material-ui/core/DialogActions';

const styles = (theme) => ({
  root: {
    margin: 0,
    padding: theme.spacing(2),
  },
  closeButton: {
    position: 'absolute',
    right: theme.spacing(1),
    top: theme.spacing(1),
    color: theme.palette.grey[500],
  },
});

const DialogContent = withStyles((theme) => ({
  root: {
    padding: theme.spacing(2),
  },
}))(MuiDialogContent);

const DialogActions = withStyles((theme) => ({
  root: {
    margin: 0,
    padding: theme.spacing(1),
  },
}))(MuiDialogActions);

export default function Modal(props) {
  return (
    <div>
      <Dialog fullWidth={true} maxWidth = {'md'} aria-labelledby="customized-dialog-title" open={props.open}>
        <DialogContent>
          { props.children }
        </DialogContent>
        <DialogActions>
          <Button autoFocus onClick={props.handleClose} color="primary">
            Return
          </Button>
        </DialogActions>
      </Dialog>
    </div>
  );
}