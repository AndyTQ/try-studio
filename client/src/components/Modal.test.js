import React from "react";
import { create } from "react-test-renderer";
import Modal from "./Modal";

describe("Submenu component", () => {
  test("Matches the snapshot", () => {
    const modal = create(<Modal><div></div></Modal>);
    expect(modal.toJSON()).toMatchSnapshot();
  });
});

