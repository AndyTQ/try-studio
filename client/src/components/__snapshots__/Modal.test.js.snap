// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Submenu component Matches the snapshot 1`] = `<div />`;
